cdap-app-fabric-tests/src/test/java/io/cdap/cdap/internal/app/runtime/distributed/ProgramTwillRunnableModuleTest.java

```
@@ -178,5 +181,10 @@ public String getName() {
     public Supplier<TwillRunnerService> getTwillRunnerSupplier() {
       return NoopTwillRunnerService::new;
     }
+
+    @Override
+    public void fetchFile(URI srcURI, File dest) throws IOException {
+      throw new IOException("Unimplemented");
```

Should throw `UnsupportedOperationException`